src/main/java/hudson/plugins/ansicolor/AnsiHtmlOutputStream.java

```diff
@@ -193,6 +194,7 @@ private void closeTagsOfTypeFGBG() throws IOException {
         }
     }
 
+    @SuppressFBWarnings("SF_SWITCH_NO_DEFAULT")
```

Comment:
This says there's a switch statement without a default case, maybe just fix that statement?